pyproject.toml

```diff
@@ -24,8 +24,9 @@ classifiers = [
   "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
-  "anyio~=3.2",
+  "anyio>=3.2",
```

Comment:
Should there be a constraint for `<5.*`?

Reply:
```suggestion
  "anyio>=3.2,<5",
```